packages/models-library/src/models_library/volumes.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+class VolumeID(str, Enum):
```

Comment:
STYLE: VolumeID suggest to me some sort of uuid or integer ... What about ``VolumeType`` or ``VolumeKind`` , ``VolumeFlavour`` , ``VolumeCategory``

Reply:
2nd. If the enum value is just used as unique identifier why not using our wonderful ``StrAutoEnum``?